src/components/download-button/__tests__/download-button.test.js

```diff
@@ -0,0 +1,68 @@
+import renderer from 'react-test-renderer';
+import { testCases } from './download-button-test-cases.js';
+
+describe('DownloadButton', () => {
+  describe(testCases.basic.description, () => {
+    let testCase;
+    let wrapper;
+    let tree;
+
+    beforeEach(() => {
+      testCase = testCases.basic;
+      wrapper = renderer.create(testCase.element);
+      tree = wrapper.toJSON();
+    });
+
+    test('renders as expected', () => {
+      expect(tree).toMatchSnapshot();
+    });
+  });
+
+  describe(testCases.png.description, () => {
+    let testCase;
+    let wrapper;
+    let tree;
+
+    beforeEach(() => {
+      testCase = testCases.png;
+      wrapper = renderer.create(testCase.element);
```

Comment:
The `png` and `zip` testcases do not have an `element` property so that's why the snapshot tests are returning null. Check out this test case for how we pull in test cases that use `component` and `props` instead of `element`:

https://github.com/mapbox/dr-ui/blob/c0df94f7677261e84f2173af361247bc0055918d/src/components/book-image/__tests__/book-image.test.js#L22-L38